policy/resolver_test.go

```diff
@@ -1451,3 +1453,54 @@ queries:
 		}
 	})
 }
+
+func TestResolve_TwoMrns(t *testing.T) {
+	b := parseBundle(t, `
+owner_mrn: //test.sth
+policies:
+- uid: policy1
+  groups:
+  - type: chapter
+    filters: "true"
+    checks:
+    - uid: check1
+      mql: asset.name == props.name
+      props:
+      - uid: name
+        mql: return "definitely not the asset name"
+    - uid: check2
+      mql: asset.name == props.name
+      props:
+      - uid: name
+        mql: return "definitely not the asset name"
+`)
```

Comment:
it might be good to also have a queries example as the data queries have their reporting jobs built differently

Reply:
Added.